data/RTTR/CAMPAIGNS/world/AFRICA.LUA

```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------------
+-- LUA-Script for AFRICA.WLD (mission of the original "Continent Campaign") --
+--                                                                          --
+-- Author: Michael Ott (v1.0)                                               --
+------------------------------------------------------------------------------
+
+
+-------------------------------- TODO -----------------------------------------
+-- EnableNextMissions()
+-- Set Portraits
+-- Set AI Agression Level
+-------------------------------------------------------------------------------
+
+
+-------------------------------- Lua Version used -----------------------------
+function getRequiredLuaVersion()
+    return 1.3
```

Comment:
I think it should be 1 instead of 1.3, guess 3 was the feature level which has been removed (in all scripts, also the roman version)